src/nodes/QskTreeNode.cpp

```diff
@@ -5,7 +5,7 @@
 
 #include "QskTreeNode.h"
 
-static constexpr auto extraFlag =
+static auto extraFlag =
```

Comment:
why no longer `constexpr` ?

Reply:
with constexpr we get this error:
```
/home/peter/dev/qskinny/src/nodes/QskTreeNode.cpp:9:5: error: integer value 33554432 is outside the valid range of values [0, 33554431] for the enumeration type 'Flag' [-Wenum-constexpr-conversion]
    9 |     static_cast< QSGNode::Flag >( QSGNode::IsVisitableNode << 1 );
      |     ^
1 error generated.
```
Any suggestion on how to fix this differently would be greatly appreciated :)

Reply:
I think this is a gray area of constexpressions not allowing undefined behavior vs. enumerator being only a set of constant on the underlying type ( e.g. int by deafult ).

Which compiler are you using?

Compromise: `static const auto extraFlag` thereby exiting `constexpr` land

https://stackoverflow.com/questions/66664841/undefined-behavior-of-constexpr-static-cast-from-int-to-scoped-enum-with-non-fix

Reply:
PS: https://eel.is/c++draft/dcl.enum#8

Reply:
PPS: see suggested

https://godbolt.org/z/n5WKde3ME

Reply:
The compiler is clang 18.0 which comes from the Emscripten SDK (needed by WebAssembly). I changed the declaration to "static const auto"